ethpm_types/abi.py

```diff
@@ -60,6 +63,7 @@ def signature(self) -> str:
 
 class ConstructorABI(BaseModel):
     type: Literal["constructor"]
+    contract_type: Optional["ContractType"] = Field(None, exclude=True)
```

Comment:
Should it be a single underscore field? That'll mean it won't get serialized (which is okay, because it will get added again during deserialization), and also makes it a little clearer thay you should understand the context of how the variable is set so that you aren't confused if it's missing

Reply:
I have been attempting to do this but Pydantic keeps failing to recognize the field when I try to set it:

```
ValueError: "EventABI" object has no field "_contract_type"
```

Reply:
Hmm, okay skip it for now I guess.